tests/utils/object_store/integration_test_gs_object_store.py

```diff
@@ -0,0 +1,98 @@
+# Copyright 2022 MosaicML Composer authors
+# SPDX-License-Identifier: Apache-2.0
+
+import time
+from pathlib import Path
+
+import pytest
+
+from composer.utils import GCSObjectStore
+
+__DUMMY_OBJ__ = '/tmp/dummy.ckpt'
+__NUM_BYTES__ = 1000
+bucket_name = 'mosaicml-composer-tests'
+
+
+@pytest.fixture
+def gs_object_store():
+    remote_dir = 'gs://mosaicml-composer-tests/streaming/'
+    yield GCSObjectStore(remote_dir)
+
+
+# Todo: call trainer to generate a ckpt and resume the train from cloud
+def test_train_resuming():
```

Comment:
still todo?

Reply:
yes, this will require more time. I will remove it and commit it with a future PR. -cc @eracah